### PR TITLE
docs(plugin): Add SAML plugin documentation

### PR DESCRIPTION
**Pull request by KongSteve**
### Summary
Add user documentation for new SAML v2.0 plugin to be released on Kong EE 3.1.0

### Reason
https://konghq.atlassian.net/browse/FT-1821

### Testing
https://deploy-preview-4734--kongdocs.netlify.app/hub/kong-inc/saml/

### CODE REVIEW

**Review by Guaris — APPROVED** [review]

**Comment by Guaris** [comment]
@kikito @mrwanny @KongSteve 
I reviewed this, let me know if you have any feedback.

File: .github/styles/kong/dictionary.txt [hunk 1]
```diff
@@ -276,6 +276,8 @@ ssl
 ssl_cert_key
 ssl_certificate
 sso
+SAML
+sectio
```
Comment (lena-larionova) [inline .github/styles/kong/dictionary.txt:280]:
```suggestion
```

File: .github/styles/kongplugins/dictionary.txt [hunk 2]
```diff
@@ -1,13 +1,16 @@
-
+SAML
+sectio
```
Comment (lena-larionova) [inline .github/styles/kongplugins/dictionary.txt:2]:
```suggestion
```

**Review by lena-larionova — APPROVED** [review]
looks like one of typos got added to the dictionaries, though it's now gone from the topic.

**Comment by KongSteve** [comment]
Should we make it clear in the docs that the plugin supports SAML 2.0 only. I know the graphic includes this but no harm in adding to the doc?

**Comment by Guaris** [comment]
> Should we make it clear in the docs that the plugin supports SAML 2.0 only. I know the graphic includes this but no harm in adding to the doc?

Yes, let's add it. I can add it in a few hours if you haven't added it already. @KongSteve

**Comment by Guaris** [comment]
@KongSteve @hanshuebner Can this be merged? It's ready from a docs perspective.

**Comment by hanshuebner** [comment]
I think this is good to go.

**Pull request merged**